.changeset/shiny-walls-press.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/backend-defaults': minor
+---
+
+**BREAKING**: Ensure that an error handler middleware exists at the end of each plugin `httpRouter` handler chain. This makes it so that exceptions thrown by plugin routes are caught and encoded in the standard error format.
```

Comment:
Hmm, what is this actually breaking? 😁 I'd expect this to at most be **BREAKING PRODUCERS**?

Reply:
Mm, maybe it was a bit strict, removed it entirely.